rpc-server/src/config.rs

```diff
@@ -57,7 +56,11 @@ pub struct ServerContext {
     /// Near rpc client
     pub near_rpc_client: crate::utils::JsonRpcClient,
     /// Blocks cache
-    pub blocks_cache: std::sync::Arc<crate::cache::RwLockLruMemoryCache<u64, CacheBlock>>,
+    pub blocks_cache:
+        std::sync::Arc<crate::cache::RwLockLruMemoryCache<u64, near_primitives::views::BlockView>>,
+    /// Chunks cache
```

Comment:
this comment is redundant imo. Alternatively, you can change it to describe the cache if it has something worth mentioning